### ISSUE TITLE
[Discussion] Combat/'Tactics'

### ISSUE THREAD

**Issue by BraedonWooding** [OrderOfThePorcupine/ProjectPorcupine#86]
While I was a strong force behind some kind of X-COM combat, as koosemose pointed out in Discord it would take away from the game experience, and so with the idea of Rimworld's combat being a little 'unfair' and too chaotic, we aimed to combine the two ideas to reach a position of improvement.  The new combat system will partake in similar style to other base building combat sections where you don't have 100% control over the characters but rather can give directions (move here, fire near here and so on), this kind of system will be replaced with a tactics system where you can set up fire zones and defensive positions along with complex logic trees (or opt into just simple 'defensive/offensive' options) to determine how your men and women 'should' act in combat, however, as usual, this won't 100% dictate their behaviour they may make rash decisions due to high stress or due to weaknesses such as being able to flank an enemy and so on giving a little bit of unease to the player without making their experience 'unfun'.

As with all discord chats, they aren't final and thus I opened this issue to see if anyone has anything they want to add to this particular point!

**Comment by koosemose** [comment]
I'm not certain quite on the feasibility of *complex* logic trees, but for knowledge of others, the general premise is to emulate setting your crews standard operating procedures, i.e. retreat when wounded or fight to the death, close bulkhead doors behind you/behind enemies, fire and retreat along set path, don't ever throw grenades, you fools/toss grenades with wild abandon, if the explosions and shrapnel don't get 'em the explosive decompression and vacuum will, etc..

**Comment by BraedonWooding** [comment]
Yep, that's complex logic trees, essentially a chain of if statements.  Most likely in the format of a grid where you can 'place a situation' (such as 'IF damaged' or 'IF retreating'...) then place reactions to that situation, incredibly simple and not really more than 2 or 3 'elements' to a single 'IF'.
